web/src/components/Header.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-return-assign */
```

Comment:
One last comment since I forgot to add this one.  Do we need this exception still?  If so, can you move it next to the spot where the issue is and change it to `// eslint-disable-next-line`?

Reply:
We don't need this any more! Yay!